packages/utils/src/promisebuffer.ts

```diff
@@ -21,22 +18,14 @@ export class PromiseBuffer<T> {
   /**
    * Add a promise to the queue.
    *
-   * @param taskProducer A function producing any PromiseLike<T>; In previous versions this used to be `@param task: PromiseLike<T>`, however, Promises were instantly created on the call-site, making them fall through the buffer limit.
```

Comment:
I would still keep this comment, even if we are losing the backward compatibility.

Reply:
Restored